app/assets/javascripts/map/services/InfoamazoniaStoryService.js

```diff
@@ -14,7 +14,7 @@ define([
 
     requestId: 'InfoamazoniaStoryService',
 
-    url: '//wri-01.cartodb.com/api/v2/sql?q=(SELECT ST_AsGeoJSON(the_geom),the_geom_webmercator, title, permalink FROM geojson union SELECT ST_AsGeoJSON(the_geom),the_geom_webmercator, title, permalink FROM geojson_1 where the_geom_webmercator is not null) union SELECT ST_AsGeoJSON(the_geom),the_geom_webmercator, title, permalink FROM geojson_2',
+    url: '//wri-01.cartodb.com/api/v2/sql?q=(SELECT ST_AsGeoJSON(the_geom),the_geom_webmercator, title, permalink FROM geojson union SELECT ST_AsGeoJSON(the_geom),the_geom_webmercator, title, permalink FROM table_4200314533 where the_geom_webmercator is not null) union SELECT ST_AsGeoJSON(the_geom),the_geom_webmercator, title, permalink FROM table_2528172694 where the_geom_webmercator is not null',
```

Comment:
mmm... the name of this table seems to be dangerous. Are you sure that you don't want to change it?